apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -501,9 +501,9 @@ public string FormatParameters(string id)
 
         public string ScopeFormatParameters(string id)
         {
-            using (_logger.BeginScope(nameof(ScopeFormatParameters) + " - {id}"))
+            using (_logger.BeginScope(nameof(ScopeFormatParameters) + " - {id}", id))
```

Comment:
Without passing a value for `{id}` this scope's state was just a `string`. This was one of the things that was making the test fail.

Reply:
Whoops, completely missed this. Thanks for fixing!